.rubocop.yml

```diff
@@ -1,9 +1,13 @@
 AllCops:
   Exclude:
     - 'lib/winrm/winrm_service.rb'
+    - 'lib/winrm/http/transport.rb'
 
 Style/Encoding:
   Enabled: true
 
 Metrics/LineLength:
   Max: 100
+
+StringLiterals:
+  Enabled: false
```

Comment:
@zenchild My preference would be one of the following:

- Use RuboCop's rule of single quotes unless interpolated.
- Consistently use double quotes (there's always an exception).

I just like consistency.